Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = () => {
+  return (
+    <div className="title">
+      <h1>Lambda School <small>@LambdaSchool &middot; <time>11 June</time></small></h1>
```

Comment:
This line looks great! Since you are only returning the h1 element, you don't need the parent div element.